new-pipeline/src/evo.js

```diff
@@ -877,10 +859,14 @@ function displayEvolution(target, lines, usePercentages, plotOptions) {
 
 }
 
-function displayEvolutions(lines, submissionLines, useSubmissionDate,
-  usePercentages) {
+function displayEvolutions(lines, useSubmissionDate, usePercentages) {
   indicate("Rendering evolutions...");
 
+  var submissionLines = lines.filter(line => line.aggregate == "Submissions");
```

Comment:
I'm not the biggest fan of magic strings. Can we at least pull this out to a constant? Or if you can think of something more clever, I'd like that too :)